README.md

```diff
@@ -13,50 +13,56 @@ installing the `ocne-catalog` chart.
-| Oracle Cloud Native Environment Application Catalog | ocne-catalog             | 2.0.0            |
-| Prometheus                                          | prometheus               | 2.31.1           |
-| Prometheus Adapter                                  | prometheus-adapter       | 0.10.0           |
-| Prometheus Node Exporter                            | prometheus-node-exporter | 1.6.1            |
-| Rook                                                | rook                     | 1.12.3           |
-| Tigera Operator with Calico 3.27                    | tigera-operator          | 1.32.4           |
-| Oracle Cloud Native Environment Dashboard           | ui                       | 2.0.0            |
+| Application                                         | Chart                       | Versions         |
+|-----------------------------------------------------|-----------------------------|------------------|
+| Cert Manager                                        | cert-manager                | 1.16.3<br>1.14.5 |
+| Cluster API Core Controller                         | core-capi                   | 1.9.4<br>1.7.1   |
+| Cluster API Kubeadm Control Plane Controller        | control-plane-capi          | 1.9.4<br>1.7.1   |
+| Cluster API Kubeadm Bootstrap Controller            | bootstrap-capi              | 1.9.4<br>1.7.1   |
+| Cluster API for Oracle Cloud Infrastructure         | oci-capi                    | 0.16.0<br>0.15.0 |
+| Cert Manager OCI Webhook                            | cert-manager-webhook-oci    | 1.1.0            |
+| CoreDNS                                             | coredns                     | 2.0.0            |
+| Dex                                                 | dex                         | 2.39.1           |
+| ExternalIP Webhook                                  | externalip-webhook          | 1.0.0            |
+| Flannel                                             | flannel                     | 2.0.0<br>0.22.3  |
+| Fluent Operator                                     | fluent-operator             | 3.2.0            |
+| Fluentd                                             | fluentd                     | 1.14.5           |
+| Grafana                                             | grafana                     | 7.5.17           |
+| Ingress Nginx                                       | ingress-nginx               | 1.9.6            |
+| Istio CRDs                                          | istio-base                  | 1.20.5           |
+| Istiod                                              | istiod                      | 1.20.5           |
+| Istio Egress Gateway                                | istio-egress                | 1.20.5           |
+| Istio Ingress Gateway                               | istio-ingress               | 1.20.5           |
+| Istio CRDs                                          | istio-base                  | 1.22.6           |
```

Comment:
Should there just be one entry for each of the Istio components, and a list of all versions supported?

Reply:
For example, see Flannel above

Reply:
Yes, you're correct. I merged those entries now.